### ISSUE TITLE
diskstats plugin not working on remotemounts in omv 5

### ISSUE THREAD

**Issue by lldsolitude** [openmediavault/openmediavault#511]
### Description of issue/question
After adding a cifs remote mount with the openmediavault-remotemount plugin, the settings fail to apply because the diskstats plugin reports the error `findfs: unable to resolve 'UUID=93a6b2fb-e822-47a5-b8cb-aa0f1ecf3a24'`.
### Steps to reproduce issue
If I add a remote mount with diskstats plugin enabled, or if I add a remote mount without diskstats plugin and then tries to install the plugin, the same error occurs:

```
findfs: unable to resolve 'UUID=93a6b2fb-e822-47a5-b8cb-aa0f1ecf3a24'
omv.local:
    Data failed to compile:
----------
    Rendering SLS 'base:omv.deploy.collectd.plugins.disk' failed: Jinja error: Command '['findfs', 'UUID=93a6b2fb-e822-47a5-b8cb-aa0f1ecf3a24']' returned non-zero exit status 1.
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/salt/utils/templates.py", line 394, in render_jinja_tmpl
    output = template.render(**decoded_context)
  File "/usr/lib/python3/dist-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 49, in top-level template code
  File "/var/cache/salt/minion/extmods/modules/omv_utils.py", line 165, in get_fs_parent_device_file
    return fs.get_parent_device_file()
  File "/usr/lib/python3/dist-packages/openmediavault/fs/__init__.py", line 163, in get_parent_device_file
    device = pyudev.Devices.from_device_file(context, self.device_file)
  File "/usr/lib/python3/dist-packages/openmediavault/fs/__init__.py", line 127, in device_file
    ['findfs', 'UUID={}'.format(self._id)]
  File "/usr/lib/python3/dist-packages/openmediavault/subprocess.py", line 63, in check_output
    return subprocess.check_output(*popenargs, **kwargs)
  File "/usr/lib/python3.7/subprocess.py", line 395, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.7/subprocess.py", line 487, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['findfs', 'UUID=93a6b2fb-e822-47a5-b8cb-aa0f1ecf3a24']' returned non-zero exit status 1.

; line 49

---
[...]
  #     "dir": "/srv/dev-disk-by-id-scsi-0QEMU_QEMU_HARDDISK_drive-scsi0-0-1-part1",
  #     "freq": 0,
  #     "fsname": "008530ff-a134-4264-898d-9ce30eeab927",
  # }
  {% if salt['mount.is_mounted'](mountpoint.dir) %}
    {% set disk = salt['omv_utils.get_fs_parent_device_file'](mountpoint.fsname) %}    <======================
    # Extract the device name from '/dev/xxx'.
    {% set _ = disks.append(disk[5:]) %}
  {% endif %}
{% endfor %}
# Append the root filesystem.
[...]
---
```

### Versions report
omv 5.0.14 with openmediavault-diskstats 5.0.1 and openmediavault-remotemount 5.0.1

**Comment by votdev** [comment]
How is the output of `blkid`?

**Comment by votdev** [comment]
IMO you've configured to mount a missing filesystem.

**Comment by lldsolitude** [comment]
This is a remote network share mounted with the openmediavault-remoteshare plugin, not a physical device, so it is not showing up in `blkid`.
I think maybe network shares in the filesystem list are supposed to be ignored by the diskstats plugin, but somehow the plugin still tries to resolve it, hence the error.

**Comment by votdev** [comment]
What is the output of `omv-confdbadm read --prettify conf.system.filesystem.mountpoint`?
I assume that the mountpoint configuration causing the problems is missing the `hidden` field. It should be `True`.

**Comment by votdev** [comment]
I close this issue because i think the problem has been fixed in openmediavault-diskstats 5.0.2, but i missed to release the latest package. See https://github.com/openmediavault/openmediavault/commit/59ac3fde14e96c72f10fff48b5cb37d38cfe49d9#diff-c6e06bf851760f305f83344cbdfeb6ce.
